src/main/kotlin/de/bund/digitalservice/useid/eidservice/EidService.kt

```diff
@@ -15,17 +14,89 @@ import org.springframework.stereotype.Service
  */
 @Service
 @ConditionalOnProperty("feature.eid-service-integration.enabled", havingValue = "true")
-class EidService constructor(config: EidServiceConfiguration) : EidService230(config) {
-    // TODO: The data attributes (for example: name, age) request should be configurable for each eService
-    public override fun getWebserviceRequest(): UseIDRequestType {
-        val request = UseIDRequestType()
-
-        val selector = OperationsRequestorType()
-        selector.givenNames = AttributeRequestType.REQUIRED
-        selector.placeOfResidence = AttributeRequestType.ALLOWED
-        selector.birthName = AttributeRequestType.ALLOWED
-        request.useOperations = selector
+class EidService private constructor(config: EidServiceConfiguration) : EidService230(config) {
+    // private fun generateUseIDRequest(requestDataGroups: List<String>): UseIDRequestType {
+    //     val request = UseIDRequestType()
+    //     val selector = OperationsRequestorType()
+    //
+    //     for (dataGroup in requestDataGroups) {
+    //         when (dataGroup) {
+    //             "DG1" -> selector.documentType = AttributeRequestType.REQUIRED
+    //             "DG2" -> selector.issuingState = AttributeRequestType.REQUIRED
+    //             "DG3" -> selector.dateOfExpiry = AttributeRequestType.REQUIRED
+    //             "DG4" -> selector.givenNames = AttributeRequestType.REQUIRED
+    //             "DG5" -> selector.familyNames = AttributeRequestType.REQUIRED
+    //             // "DG6" ->
+    //             "DG7" -> selector.academicTitle = AttributeRequestType.REQUIRED
+    //             "DG8" -> selector.dateOfBirth = AttributeRequestType.REQUIRED
+    //             "DG9" -> selector.placeOfBirth = AttributeRequestType.REQUIRED
+    //             "DG10" -> selector.nationality = AttributeRequestType.REQUIRED
+    //             // "DG11" ->
+    //             // "DG12" ->
+    //             "DG13" -> selector.birthName = AttributeRequestType.REQUIRED
+    //             // "DG14" ->
+    //             // "DG15" ->
+    //             // "DG16" ->
+    //             "DG17" -> selector.placeOfResidence = AttributeRequestType.REQUIRED
+    //             // "DG18" ->
+    //             "DG19" -> selector.residencePermitI = AttributeRequestType.REQUIRED
+    //             // "DG20" ->
+    //             // "DG21" ->
+    //             // "DG22" ->
+    //             else -> {
+    //                 throw IllegalStateException("Invalid data group for this eService")
+    //             }
+    //         }
+    //     }
+    //
+    //     request.useOperations = selector
+    //     return request
+    // }
+    //
+    fun getTcToken(refreshUrl: String, requestDataGroups: List<String>): TCTokenType {
+        return getTcToken(refreshUrl)
+    }
+    //     val request = webserviceRequest
```

Comment:
What are the following lines for?  why is it commented out? 
Logic needed at getIdentity endpoint?